web-www/next.config.js

```diff
@@ -13,6 +13,13 @@ const nextConfig = {
   },
   async redirects() {
     return [
+      {
+        source: '/nct',
+        destination:
+          'https://www.notion.so/regennetwork/A-Guide-to-Nature-Carbon-Ton-NCT-8204ea9d20d0436281f49b8fd1b3fbd2',
+        permanent: false,
+        basePath: false,
+      },
```

Comment:
it seems like this got mixed up with other changes from another PR

Reply:
whoops, just had to rebase